crates/transaction-pool/Cargo.toml

```diff
@@ -76,45 +76,45 @@ serde_json.workspace = true
 [features]
 default = ["serde"]
 serde = [
-	"dep:serde",
-	"reth-execution-types/serde",
-	"reth-eth-wire-types/serde",
-	"reth-provider/serde",
-	"alloy-consensus/serde",
-	"alloy-eips/serde",
-	"alloy-primitives/serde",
-	"bitflags/serde",
-	"parking_lot/serde",
-	"rand?/serde",
-	"smallvec/serde",
-	"reth-primitives-traits/serde",
-	"revm-interpreter/serde",
-	"revm-primitives/serde"
+    "dep:serde",
+    "reth-execution-types/serde",
+    "reth-eth-wire-types/serde",
+    "reth-provider/serde",
+    "alloy-consensus/serde",
+    "alloy-eips/serde",
+    "alloy-primitives/serde",
+    "bitflags/serde",
+    "parking_lot/serde",
+    "rand?/serde",
+    "smallvec/serde",
+    "reth-primitives-traits/serde",
+    "revm-interpreter/serde",
+    "revm-primitives/serde",
 ]
 test-utils = [
-	"rand",
-	"paste",
-	"serde",
-	"reth-chain-state/test-utils",
-	"reth-chainspec/test-utils",
-	"reth-primitives/test-utils",
-	"reth-provider/test-utils",
-	"reth-primitives-traits/test-utils",
+    "rand",
+    "paste",
+    "serde",
+    "reth-chain-state/test-utils",
+    "reth-chainspec/test-utils",
+    "reth-primitives/test-utils",
+    "reth-provider/test-utils",
+    "reth-primitives-traits/test-utils",
 ]
 arbitrary = [
-	"proptest",
-	"reth-primitives/arbitrary",
-	"proptest-arbitrary-interop",
-	"reth-chainspec/arbitrary",
-	"reth-eth-wire-types/arbitrary",
-	"alloy-consensus/arbitrary",
-	"alloy-eips/arbitrary",
-	"alloy-primitives/arbitrary",
-	"bitflags/arbitrary",
-	"reth-primitives-traits/arbitrary",
-	"smallvec/arbitrary",
-	"revm-interpreter/arbitrary",
-	"revm-primitives/arbitrary"
+    "proptest",
+    "reth-primitives/arbitrary",
+    "proptest-arbitrary-interop",
+    "reth-chainspec/arbitrary",
+    "reth-eth-wire-types/arbitrary",
+    "alloy-consensus/arbitrary",
+    "alloy-eips/arbitrary",
+    "alloy-primitives/arbitrary",
+    "bitflags/arbitrary",
+    "reth-primitives-traits/arbitrary",
+    "smallvec/arbitrary",
+    "revm-interpreter/arbitrary",
+    "revm-primitives/arbitrary",
```

Comment:
inb4 this is changed again

can't take this anymore 😭

Reply:
it is what it is